src/java/com/puppetlabs/ssl_utils/ExtensionsUtils.java

```diff
@@ -739,7 +739,7 @@ private static AuthorityKeyIdentifier mapToAuthorityKeyIdentifier(
                             pubKey.getEncoded());
 
             DigestCalculator digCalc =
-                    new BcDigestCalculatorProvider().get(
+                    new JcaDigestCalculatorProviderBuilder().build().get(
```

Comment:
Do we not need to call `setProvider` on this? This is my own lack of understanding for sure, but if we do not set the provider does it use a default one? If it uses a default one is that for sure going to be the FIPS one? And if those things are true, do we need to set the provider for `JcaContentVerifierProviderBuilder` in `SSLUtils`?

Reply:
We could explicitly set the provider here, but I was intending for that to happen by the JCA-JCE security properties (see PE-26658)

Reply:
If we set the security provider via the properties file, why do we have to explicitly set the `JcaContentVerifierProviderBuilder` in `SSLUtils`?

Reply:
I think I'm starting to get your question - we are calling setProvider elsewhere. The properties file is going to make any dependencies we use do the right thing, and in here, we are calling the appropriate provider directly.

Reply:
So this class is defined in both bc and bcfips and does the correct thing in both without having to specify which provider to use?

Reply:
Yes.

`import org.bouncycastle.operator.jcajce.JcaDigestCalculatorProviderBuilder;`

`BcDigestCalculatorProvider` does not exist in the bc-fips jar, but `JcaDigestCalculatorProviderBuilder` exists in both.